tests/ICU4N.Tests/Dev/Test/Util/ULocaleTest.cs

```diff
@@ -4259,7 +4259,12 @@ public void TestCLDRVersion()
             TestFmwk.Logln("uloc_getCLDRVersion() returned: '" + cldrVersion + "'");
 
             // why isn't this public for tests somewhere?
-            Assembly testLoader = typeof(ICUResourceBundleTest).GetTypeInfo().Assembly;
+            Assembly testLoader =
+#if FEATURE_TYPEEXTENSIONS_GETTYPEINFO
+            typeof(ICUResourceBundleTest).GetTypeInfo().Assembly;
+#else
+            typeof(ICUResourceBundleTest).Assembly;
```

Comment:
Please indent this 4 spaces so it is clear it is a line continuation, not a new line.

Reply:
Okay, i will fix this in the next commit.